KarhooUISDK/CommonUI/UIDimens.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  UIDimens.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 05.01.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+// Note: When updating the UI, aproximate the commented values to the ones before each comment. E.g.: replace any spacing of value 17 with .standard (which is 16)
+public struct UIDimens {
+    public struct Spacing {
```

Comment:
It's good idea to add comments with value used in the constant, so user (another developer) do not need to open the source file.
For example:
```
public struct Spacing { 
    /// 1
    public static let xxxSmall: CGFloat = 1
    (...)
}
```

Reply:
This applies to all file